components/rmrk/Gallery/Holder/Holder.vue

```diff
@@ -220,13 +238,26 @@ type BaseTableRow = {
   Amount: number
 }
 
-@Component({ components })
-export default class Holder extends mixins(ChainMixin, KeyboardEventsMixin) {
+@Component({
+  filters: {
+    percentageFilter: (value) => {
```

Comment:
This seems to be obsolete :) 

Great work with the `toPercent` filter

Reply:
Oh, I have used `toPercent` but forget to delete this one ; )